sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -154,16 +154,22 @@ export class ApplicationApi extends HttpBaseClient {
   }
 
   /**
-   * API Client for applications of a student.
-   * @param studentId
+   * API Client to get student applications.
```

Comment:
IMO, i believe we can have 2 API calls, atleast in the web service layer, We can discuss on this further.

Reply:
it was separate before, logic inside the function is the same, so I got code duplication in sonarqube

Reply:
@guru-aot did you mean separate API call for student and applications?